Migrate to FW SDK v6.0.0 (#38)
loopsocial/firework_android_sdk_examples: README.md

* [Feed Resources](#feed-resources-example-app)
* [Single-Host Livestream](#single-host-livestream-example-app)
* [Multi-Host Livestream](#multi-host-livestream-example-app)
* [Shopping](#shopping-example-app)
* [Share Link](#share-link-example-app)
* [Picture In Picture](#picture-in-picture-example-app)
* [Jetpack Compose](#jetpack-compose-example-app)

**Read more about the integration here** https://docs.firework.tv/

---

#### View Options example app

> This example app demonstrates all possible view options that can be used for the `VideoFeedView` initialization.

Reviewer: ```suggestion
This example app demonstrates all possible view options that can be used for the `FwVideoFeedView` initialization.
```

> In this example app, the `VideoFeedView` is initialized with different feedResources: Discovery, Playlist, Channel, Dynamic Content.

Reviewer: ```suggestion
In this example app, the `FwVideoFeedView` is initialized with different feedResources: Discovery, Playlist, Channel, Dynamic Content.
```

> In this example app, the `VideoFeedView` is initialized with different feedResources: Discovery, Playlist, Channel, Dynamic Content.

Reviewer: ```suggestion
In this example app, the `FwVideoFeedView` is initialized with different feedResources: Discovery, Playlist, Channel, Dynamic Content.
```

2. `ShareLinkActivity` - handles the deep link and opens the shared video.
Also this example demonstrates how to replace the base url of the share video with custom one.

---
## Picture-in-picture example app

> In this example app, the `VideoFeedView` is initialized for switch to picture-in-picture mode if user in in player and tries to leave the app, or when user presses the down shevrun on player, or when related public API is being called.

Reviewer: ```suggestion
In this example app, the `FwVideoFeedView` is initialized for switch to picture-in-picture mode if user in in player and tries to leave the app, or when user presses the down chevron on player, or when related public API is being called.
```

> In this example app, the `VideoFeedView` is initialized for switch to picture-in-picture mode if user in in player and tries to leave the app, or when user presses the down shevrun on player, or when related public API is being called.

Reviewer: ```suggestion
In this example app, the `FwVideoFeedView` is initialized for switch to picture-in-picture mode if user in in player and tries to leave the app, or when user presses the down chevron on player, or when related public API is being called.
```

[Picture-in-picture example app](picture_in_picture )

> | PiP Feed                                              | Player screen in PiP mode                                    |

Reviewer: ```suggestion
| PIP Feed                                              | Player screen in PIP mode                                    |
```

| ----------------------------------------------------- | ------------------------------------------------------------ |
| ![pip Screenshot](picture_in_picture/Screenshot1.png) | ![player in pip mode Screenshot](picture_in_picture/Screenshot2.png) |

---
## Jetpack Compose example app

> In this example app, the `VideoFeedView` is initialized for showing a discovery feed using Jetpack Compose.

Reviewer: ```suggestion
In this example app, the `FwVideoFeedView` is initialized for showing a discovery feed using Jetpack Compose.
```
